### ISSUE TITLE
Opening a ipynb file in vscode

### ISSUE THREAD

**Issue by nscsekhar** [DonJayamanne/vscodeJupyter#52]
### Environment data
VS Code version:1.11.1
Jupyter Extension version:4.2.1
OS and version:OSX Sierra

### Logs
Output from ```Jupyter``` output panel
None

Output from ```Console window``` (Help->Developer Tools menu)
None

### Actual behavior
When I try to open a ipynb file, it opens as a raw json like data. 

### Expected  behavior
I would like to see it as cells and text. 

I feel the web interface for ipynb is not great for step wise edit and debugging. I think vscode jupyter is really great extension but it works only on cells (atleast the examples I have seen). Is there a way to open existing ipynb files in vscode in a way to run and debug?

Thanks..

### Steps to reproduce:
Open any ipynb file using vscode after installing the jupyter extension.
-
-

**Comment by DonJayamanne** [comment]
Ipynb files are not supported by this extension. There are no immediate plans to support this either.

**Comment by neilsustc** [comment]
Could this be reopened as a feature request? It is a really great feature.

**Comment by eromoe** [comment]
Yeah, with .ipynb support, you can share your notebook with others more easily. 
At least,  please support convert bettween `.ipynb`  and `.py` .

**Comment by sajaysurya** [comment]
A 2-way conversion between .ipynb and .py (suitable for this extension) would be very useful. Please implement this enhancement, if possible.

**Comment by jarandaf** [comment]
It should be relatively easy to accomplish using [nbconvert](https://github.com/jupyter/nbconvert).

**Comment by tigerhawkvok** [comment]
I was just looking for this and ran across this issue, so let me chime in with support.

@jarandaf , maybe submit a PR? I'm new to notebooks so I'd be digging in the dark.

**Comment by harrypotter0** [comment]
@tigerhawkvok  Any update on this issue ?

**Comment by kabukunz** [comment]
Yes please! It'd be very handy!

**Comment by mwouts** [comment]
I plan to work on a script exporter/importer for Jupyter notebooks, compatible with the Jupyter extension for VS code, at https://github.com/mwouts/jupytext/issues/59. Please review the specifications there if you are interested. Thanks.

**Comment by mwouts** [comment]
Hello everyone. I just published a new RC for Jupytext. Jupytext can now convert double percent scripts to Jupyter notebooks, and back. Would you like to test and provide feedback?

Install the rc with
```
pip install jupytext==0.7.0rc0
```

Corresponding documentation (how to use the command line interface, or how to setup Jupyter to read or save notebooks to double percent files) is the [v0.7.0 branch](https://github.com/mwouts/jupytext/blob/v0.7.0/README.md).

Feedback is welcome!
